package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
     "build-doc": "node scripts/build.js && cp build/index.html build/404.html",
+    "build-doc:netlify": "npm run build && NETLIFY=true npm run build-doc",
```

Comment:
Am I missing something here?  This new script has been added, but it doesn't appear to ever get called.  Is there some background black magic happening?

Reply:
@greg-a-smith Yeah this pr is a little confusing without context, sorry about that. I also did it a little out of order, so the comments won't show until the next pr. 

The magic is happening on the netlify dashboard. We were using a `netlify.toml` file in the fundamental repo as that overrides any settings in the dashboard, which we didn't have access to at the time. 

<img width="1266" alt="Screen Shot 2019-04-12 at 7 34 07 AM" src="https://user-images.githubusercontent.com/29607818/56045132-7cad8400-5cf5-11e9-847c-d0bc4298cb85.png">

Reply:
Got it, thanks.  Although we don't _have to_, I'm curious if it's worth just using a `netlify.toml` file in each repo to reduce the "magic".